openpype/hosts/zbrush/hooks/force_startup_script.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+"""Pre-launch to force zbrush startup script."""
+import os
+from openpype.hosts.zbrush import ZBRUSH_HOST_DIR
+from openpype.lib import get_openpype_execute_args
+from openpype.lib.applications import PreLaunchHook, LaunchTypes
+
+
+class ForceStartupScript(PreLaunchHook):
```

Comment:
No idea how uniqueness works of prelaunch hooks across different addons/hosts. But should this class name include `Zbrush` or whatever to ensure uniqueness?

Reply:
`ZbrushStartupLaunchHook` or whatever could be fine, yes.